ui.apps/src/main/content/jcr_root/apps/hclecomm/config.author.qa/com.hcl.ecomm.core.services.impl.ProductServiceImpl.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
@DeepaliAgrwl : why havent you added configuration for config.author.dev ?

Reply:
added